inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_reshape.cpp

```diff
@@ -23,6 +23,9 @@ void dynamicToStaticShapeReshape(std::shared_ptr<ngraph::Node> target) {
                      target->get_friendly_name(), target->get_type_info(), ngraph::vpu::op::DynamicShapeResolver::type_info, 0);
 
     const auto reshape = std::dynamic_pointer_cast<ngraph::opset3::Reshape>(target);
+    VPU_THROW_UNLESS(reshape != nullptr,
+        "Cannot cast operation {} of type {} to {}",
+        target->get_friendly_name(), target->get_type_info(), ngraph::opset3::Reshape::type_info);
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/2142#issuecomment-690105396

@Maxim-Doronin please take a look

Reply:
Oh, both fixes are similar. 
@ilyachur, may we decide to remove fixes for `vpu/common/src/ngraph` from one of PR's not to conflicts

Reply:
Ok, cool. I will remove all changes in VPU part

Reply:
Removed